webapp/channels/src/actions/views/rhs.ts

```diff
@@ -520,11 +514,14 @@ export function toggleRhsExpanded() {
     };
 }
 
-export function selectPost(post: Post) {
+export function selectPost(post: Post, previousRhsState?: RhsState) {
+    performance.mark('PostList#postSelected');
```

Comment:
Yep, I can definitely do that. Since we want to be adding more of these, it'll be good to have them all in one place too

Reply:
I'm actually going to move that to a followup PR because I want to move the existing measures and labels into constants as well which is outside of what I want to change in this PR

Reply:
I ended up not doing that because it ended up not being worth it to split into a separate PR